client/src/pages/Login.js

```diff
@@ -0,0 +1,127 @@
+import { makeStyles } from '@material-ui/core/styles'
+import { Grid, Paper, Typography, TextField, Button } from '@material-ui/core';
+
+function Login() {
+    const classes = useStyles();
+
+    const form = {
+        email: '',
+        password: '',
+    }
+
+    const [formValues, setFormValues] = useState(form)
+    const { email, password } = formValues
+
+    const onSubmit = e => {
+        e.preventDefault();
+        // TODO
+    }
+
+    const handleChange = e => {
+        const target = e.target
+
+        setFormValues(prevState => ({
+            ...prevState,
+            [target.name]: target.value
+        }))
+    }
+
+    return (
+        <div className={classes.root}>
+            <Navbar type="login">Don't have an account?</Navbar>
+            <Grid container spacing={0} className={classes.gridRoot} justify='center' alignItems='center'>
+                <Grid item xs={6}>
+                    <Paper className={classes.Paper}>
+                        <Typography className={classes.FormTitle}>Signup</Typography>
+                        <form action="/login" method="POST" onSubmit={onSubmit}>
+                            <Grid item container spacing={2} justify="center" alignItems='center'>
+                                <Grid item xs={10}>
+                                    <TextField
+                                        className={classes.TextFields}
+                                        required
+                                        name="email"
+                                        label="Your Email"
+                                        type="email"
+                                        variant="outlined"
+                                        value={email}
+                                        onChange={handleChange}
+                                    />
+                                </Grid>
+                                <Grid item xs={10}>
+                                    <TextField
+                                        className={classes.TextFields}
+                                        required
+                                        name="password"
+                                        label="Password"
+                                        type="password"
+                                        variant="outlined"
+                                        value={password}
+                                        onChange={handleChange}
+                                    />
+                                </Grid>
+                                <Grid item xs={4}>
+                                    <Button className={classes.LoginButton} type="submit">Login</Button>
+                                </Grid>
+                            </Grid>
+                        </form>
+                    </Paper>
+                </Grid>
+            </Grid>
+        </div>
+    );
+}
+
+const useStyles = makeStyles(theme => ({
```

Comment:
Let's consistently decide to put useStyles above or below the component. Just choose one

Reply:
Moved it down